hyp3_rtc_gamma/__main__.py

```diff
@@ -96,7 +98,14 @@ def get_download_url(granule):
 def download_file(url):
     print(f"\nDownloading {url}")
     local_filename = url.split("/")[-1]
-    with requests.get(url, stream=True) as r:
+    session = Session()
+    retries = Retry(
+        total=3,
+        backoff_factor=10,
+        status_forcelist=[429, 500, 503, 504],
+    )
+    session.mount('https://', HTTPAdapter(retries=retries))
```

Comment:
might as well mount http as well:
`session.mount('http://', HTTPAdapter(retries=retries))`
or a single, shorter prefix, but this doesn't seem to be how people do it:
`session.mount('http', HTTPAdapter(retries=retries))`

Reply:
or pull the scheme out of `url`, but that seems overkill

Reply:
and turns out it's `max_retries=retries`, my bad on the typo